.github/workflows/pull-requests.yml

```diff
@@ -0,0 +1,49 @@
+name: PR Validation
+
+on:
+  push:
+    branches: [ "main", "development" ]
```

Comment:
just calling out we don't have a development branch yet.

Reply:
Indeed - but I took that basically from the preexisting `continuous-delivery-pipeline.yml`. I thought it was there for a reason.

Reply:
I think it was just a default template that was copied over - but doesn't hurt to just leave this is in for now.. all good 😄